src/pybind/rados/rados.pyx

```diff
@@ -4009,6 +4009,110 @@ returned %d, but should return zero on success." % (self.name, ret))
         if ret < 0:
             raise make_ex(ret, "Ioctx.rados_lock_exclusive(%s): failed to set lock %s on %s" % (self.name, name, key))
 
+    def break_lock(self, key: str, name: str, client: str, cookie: str):
+
+        """
+        Release a shared or exclusive lock on an object , which was taken by the
+        specified client.
+
+        :param key: name of the object
+        :param name: name of the lock
+        :param client: the client currently holding the lock
+        :param cookie: cookie of the lock
+
+        :raises: :class:`TypeError`
+        :raises: :class:`Error`
+        """
+        self.require_ioctx_open()
+
+        key_raw = cstr(key, 'key')
+        name_raw = cstr(name, 'name')
+        client_raw = cstr(client, 'client')
+        cookie_raw = cstr(cookie, 'cookie')
+
+        cdef:
+            char* _key = key_raw
+            char* _name = name_raw
+            char* _client = client_raw
+            char* _cookie = cookie_raw
+
+        with nogil:
+            ret = rados_break_lock(self.io, _key, _name, _client, _cookie)
+        if ret < 0:
+            raise make_ex(ret,
+                          "Ioctx.rados_break_lock(%s): failed to break lock %s "
+                          "on %s for client %s" % (self.name, name, key, client))
+
+    def list_lockers(self, key: str, name: str):
+
+        """
+        List clients that have locked the image and information
+        about the lock.
+
+        :param key: name of the object
+        :param name: name of the lock
+
+        :returns: dict - contains the following keys:
+
+                  * ``tag`` - the tag associated with the lock (every
+                    additional locker must use the same tag)
+                  * ``exclusive`` - boolean indicating whether the
+                     lock is exclusive or shared
+                  * ``lockers`` - a list of (client, cookie, address)
+                    tuples
+
+        :raises: :class:`TypeError`
+        :raises: :class:`Error`
+        """
+
+        self.require_ioctx_open()
+
+        key_raw = cstr(key, 'key')
+        name_raw = cstr(name, 'name')
+
+        cdef:
+            char* _key = key_raw
+            char* _name = name_raw
+            int exclusive = 0
+            size_t addrs_size = 512, tag_size = 512
+            size_t clients_size = 512, cookies_size = 512
```

Comment:
Nit: Keep these declarations separate and in the same order as respective char pointers/as they go in the API:
```suggestion
            size_t tag_size = 512
            size_t clients_size = 512
            size_t cookies_size = 512
            size_t addrs_size = 512
```